### PR TITLE
Default --root option to ENV BUILDAHROOT

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: cmd/buildah/addcopy.go [hunk 1]
```diff
@@ -14,7 +15,8 @@ var (
 		},
 		cli.StringFlag{
 			Name:  "root",
-			Usage: "root `directory` of the working container",
+			Usage: "root `directory` of working container, Default: EnvVar(BUILDAHROOT)",
+			Value: os.Getenv("BUILDAHROOT"),
```
Comment (nalind) [inline cmd/buildah/addcopy.go]: I think you can set `EnvVar: "BUILDAHROOT"` here and let the cli package handle the rest.

**Comment by rhatdan** [comment]
Very cool

**Comment by nalind** [comment]
LGTM pending integration tests.

**Comment by nalind** [comment]
Merging.

**Comment by nalind** [comment]
[[Buildah] (8) buildah CLI overhaul](https://trello.com/c/LULizZlk/442-buildah-8-buildah-cli-overhaul)

**Pull request merged**